packages/wasm/crate/src/keys.rs

```diff
@@ -21,10 +20,7 @@ use crate::utils;
 /// function will additionally require downloading the proving key parameter `.bin`
 /// file for each key type.
 #[wasm_bindgen]
-pub fn load_proving_key(parameters: JsValue, key_type: &str) -> WasmResult<()> {
-    // Deserialize JsValue into Vec<u8>.
-    let parameters_bytes: Vec<u8> = Uint8Array::new(&parameters).to_vec();
-
+pub fn load_proving_key(key: Vec<u8>, key_type: &str) -> WasmResult<()> {
```

Comment:
This is definitely a good example of where bytes are needed anyway. It doesn't make sense to serialize through JsValue. However, I don't think this fits in all cases. Will comment on https://github.com/penumbra-zone/web/issues/845 as I make my way through it.